packages/typescript/lib/quickstart/runTsc.ts

```diff
@@ -12,6 +12,7 @@ export function runTsc(
 	options: string[] | {
 		extraSupportedExtensions: string[];
 		extraExtensionsToRemove: string[];
+		proxyTypescript?: boolean;
```

Comment:
Am I reading the code correctly that it will default to `true` if not set? If so then that seem confusing. You could then rename it to mean the opposite, like `disableProxyTypescript`.